mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -30,3 +34,21 @@ def csv_stdlib(fp):
         })
     rows = [row for row in reader]
     return columns, rows
+
+
+def detect_quoted_fields(dialect, data):
```

Comment:
Is this necessary, or can we take advantage of the quoting options in the `csv` module? https://docs.python.org/2/library/csv.html#csv.QUOTE_ALL